Header header
visualization_msgs/Marker[] markers
float32[] confidences
